### DISCUSSION TITLE
add resources requests and limits recommendations

### DISCUSSION THREAD

**Discussion by yardenasadosa** [apache/airflow#37068] [General]
When using K8S, requests and limits are often needed. The Example in the documentation should have recommended values for each component for a minimal setup, medium, large, etc.
at the moment, using the values in the example will not work.

**Comment by potiuk** [comment]
Sure. Good idea. Please experiment with it and add better examples in the form of Pull Request.

This is super easy. Click "Suggest a change on this page" where you think it should be added and PR where you will be able to add such documentation will be added for you. Like almost 2800 other contributors, who created airflow, and many of them who just contributed such documentation change - you have a chance to give back fort the free software you get and become a contributor.

Can we count on your help here @yardenasadosa ?
